DataFormats/TrackingRecHitSoA/src/classes.cc

```diff
@@ -2,6 +2,8 @@
 #include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsSoA.h"
 #include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
 
+using namespace reco;
+
 SET_PORTABLEHOSTCOLLECTION_READ_RULES(PortableHostCollection<TrackingRecHitLayout<pixelTopology::Phase1>>);
 SET_PORTABLEHOSTCOLLECTION_READ_RULES(PortableHostCollection<TrackingRecHitLayout<pixelTopology::Phase2>>);
 SET_PORTABLEHOSTCOLLECTION_READ_RULES(PortableHostCollection<TrackingRecHitLayout<pixelTopology::HIonPhase1>>);
```

Comment:
Does it work if you use `reco::TrackingRecHitLayout<...>` explicitly instead of adding `using namespace reco;` ?

Reply:
Yes, it works. Should I remove the `using namespace reco` from there? [`TrackSoA`](https://github.com/borzari/cmssw/blob/AlpakavsCUDA/DataFormats/TrackSoA/src/classes.cc#L5-L8) has the same thing, should it be changed as well?

Reply:
Yes, please.